### ISSUE TITLE
Duplicate request - changes are linked with the original

### ISSUE THREAD

**Issue by HarpunDK** [a85/POSTMan-Chrome-Extension#7350]
**Describe the bug**

**To Reproduce**
This is in the postman client. Most recent update and earlier as well.

1. Open an existing request or create a new request
2. Set a name to the request (click save if you like)
3. Right click on the request-tab and chose "Duplicate tab"
4. Set the name/description of the new request
5. Now both tabs adapts the new value, when set on the duplicated tab. Also the name changes under "Collections", if saved, without saving the changes. 

**Expected behavior**
In my opionon as a user, I would expect the new tab to be a "copy" of the original. When updating the clone, the original should not be affected/changed. I would point the are linked, which they should not. Should be copy  - sorry if I am wrong.

Remark: If you reproduce this bug and renames the original, the tabs acts like copies as expected. This being a workaround. 

**App information (please complete the following information):**
 - Postman Version [7.7.2]

**Comment by ryankeener** [comment]
@HarpunDK I've been frustrated by this as well and I agree that it's unexpected behavior. However, I just learned that if you 'duplicate' a request from the sidebar instead of a tab it will create a copy that is not linked to the original. Having the same command do two different things is bad UX but hopefully that workaround helps you out.

More info here: #4938

**Comment by eduardboamba** [comment]
On version v7.15.0 and issue is still there. Thanks @ryankeener for the workaround.

**Comment by lesmuc** [comment]
On version v7.18.0 and issue is still there.

**Comment by IonelLupu** [comment]
and still an issue on V7.21 ...

**Comment by b-hayes** [comment]
super annoying. How do I save to copies of the request with different settings?

**Comment by cognivator** [comment]
Still an issue (with duplicating Collections) in v7.27.x...
Duplicate to a copy, remove the original collection - they both disappear.
Also, the duplicates cannot be moved to a different workspace.

**Comment by elieobeid7** [comment]
This issue should be a top priority postman team. It's unacceptable that such a grave issue lives for over a year. This issue alone is enough reason for me or anyone to look for an alternative to postman.

**Comment by bzifkin44** [comment]
Agreed with the above comments. Very frustrating and makes no sense that clicking "Duplicate" in different parts of the UI would have different effects

**Comment by BhanuHoysala** [comment]
Issue is not yet resolved yet, Postman `v7.34.0 `😞

**Comment by Andreas-Scholz** [comment]
The issue still exists in `v7.36.1`

**Comment by dhruvrnaik** [comment]
Why is this not a top priority!

**Comment by worlebird** [comment]
Issue still exists. Please fix.

**Comment by douchen** [comment]
still on `v8.0.6`

**Comment by fatmalama** [comment]
wow this is really next level terrible UX

**Comment by antonylineesh1** [comment]
Issue still exists on  v8.3.1

**Comment by chenkunyun** [comment]
Works perfectly fine on v8.6.2.
This issue can be closed now

**Comment by IonelLupu** [comment]
@chenkunyun I am on 8.7.0 and the issue still exists: duplicating a saved request and updating the duplicated one will make the original request get updated as well (eg. changing the name of the request)

**Comment by masonmcelvain** [comment]
I am on 8.9.1 and this issue still exists.
Using the postman desktop client on macOS Big Sur 11.4.

**Comment by hongminhcbg** [comment]
This is very bad UX, please fix this issue

**Comment by fanjiangqi** [comment]
Issue still exists on v8.11.1,in win10

**Comment by MarkRK** [comment]
Issue still present on v9.0.3

**Comment by Hashimlokasher** [comment]
Issue still present on v9.11.0

**Comment by Healios** [comment]
Just want to chime in, this is still terrible.

**Comment by CmdrBeavis** [comment]
Yeup, _still_ seeing this in `9.15.2`!!!

FACE. PALM.

**Comment by AbdouTanta** [comment]
Yup this is still an issue! Frustrating.

**Comment by sabribrn** [comment]
Issue still present on v9.24.1

**Comment by kticka** [comment]
I think my problem is related to this issue. When generating documentation I see the wrong URLs on duplicated requests. They are taken from the original request.

**Comment by john-aws** [comment]
It might be helpful if someone at Postman could explain to the suffering devs here why the duplicate feature actually works the way it does. Why does it link the two requests? What is the primary use case for this (unexpected) behavior? Right now it's just baffling and has been for 4 years.

**Comment by dminGod** [comment]
@vkaegis  @shamasis - been more than 2 years+ on this.. 🤷🏼‍♂️ 

If a person makes a duplicate request to make another similar request so they don't have to do the work of making the request again it will get made, but when you modify it your original request will get messed up.

pls fix, this is basic functionality.

**Comment by Gonzalo2683** [comment]
Issue still present on v10.8.0

**Comment by chriscallan** [comment]
Still alive on 10.13.  This makes the software gods very sad :-(
I can really see why I've never met anyone that actually pays for Postman.  With straightforward bugs that languish for 5+ years...   I wouldn't pay for a tool with "support" like that either

**Comment by gitJoe42** [comment]
Still present in 10.16.3

**Comment by nigiraudo** [comment]
longest running gag ever 🤣 
Still there in v10.18 (09/2023)

**Comment by CmdrBeavis** [comment]
> longest running gag ever 🤣 Still there in v10.18 (09/2023)

Right?! I just got nailed by this two days ago. :face_palm:

**Comment by theelderbeever** [comment]
Might be fixed as of Version 10.18.10!!!!

**Comment by gitJoe42** [comment]
> Might be fixed as of Version 10.18.10!!!!

Praise the Lord!

**Comment by amarpreetsingh29** [comment]
Please fix this, it's really bad

**Comment by gitJoe42** [comment]
Not fixed in 10.19.2 :(

**Comment by jaykeerti** [comment]
v10.20
still persists :-(

**Comment by bearded-giant** [comment]
Funny that this is still a thing.  Moreover, why on earth would anyone want to duplicate a request only to have the two linked?  To preserve the output of the original?  This makes zero sense ya'll.

**Comment by CmdrBeavis** [comment]
> Funny that this is still a thing. Moreover, why on earth would anyone want to duplicate a request only to have the two linked? To preserve the output of the original? This makes zero sense ya'll.

Could not agree more.

**Comment by alcalyn** [comment]
I was looking for a feature, but I found this bug that could be my feature !

I want to create links to another requests, for documentation purpose.
I.e, I have multiple requests organized by components, and a folder for a given feature. In this feature folder, I want to create links to requests that are relevant to this feature.

**Comment by chriscallan** [comment]
@alcalyn your feature idea sounds like a good one, but it ain't this bug.
This bug essentially gives you 2 requests that share all their information, so it isn't 2 requests really.  Rather 1 request with 2 UI components.  Change one request and the other has the changes instantly.
If this bug was in any way useful, I could understand your excitement, but this seems to be a simple case of reusing the pointer for UI components, but likely more complex since this bug has been lingering for many years at this point.

**Comment by rajaravivarma-r** [comment]
Still not resolved.
